Common.mk

```diff
@@ -284,6 +284,10 @@ define BINARY_TARGET_BODY
 
 endef
 
+define GET_CLONE_URL
```

Comment:
can we call this up around https://github.com/aws/eks-anywhere-build-tooling/pull/377/files#diff-6daeb9c300098022af9eb1c5f69e52ffbd840490fd9945ac9c3e2d02ad838ab7R43

Reply:
Done, refactored it a bit.